test/language/statements/for-of/head-await-using-bound-names-fordecl-tdz.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.6.4.12_S2
```

Comment:
We're not using `es6id` anymore so this should be replaced by `esid` with the section ID anchor. (Please check in several other tests as well)